MuEnvironment/Omnicache.py

```diff
@@ -87,6 +87,12 @@ def Contains_name(self, name):
                 return True
         return False
 
+    def NameForUrl(self, url):
```

Comment:
Nit picky thing: most of the functions on this class are Verb_description. Maybe rename this to GetNameForUrl? Just a suggestion

Reply:
Fixed.